src/pages/SignupPage.jsx

```diff
@@ -1,6 +1,38 @@
 import React from 'react';
 import { Layout } from '../components';
+import { Grid } from '@mui/material';
+import { Input } from '../components/Inputs';
+import { CustomButton } from '../components/Button/CustomButton';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+import { SignUpTheme, SingUpTeme } from '../styles/themes/CustomSingUpPage';
+import { width } from '@mui/system';
```

Comment:
keep convention of import elements
first import from node_modules
empty line
import local files